src/main/java/by/itech/lab/supplier/controller/UserController.java

```diff
@@ -97,6 +98,12 @@ public void changeActive(@PathVariable Long id, @RequestBody boolean status) {
         userService.changeActiveStatus(id, status);
     }
 
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
Which status is used by default? and anyone will be able to change the status to anyone?